### PR TITLE
PCRM-0000 Add Account Level AWS Configs

### PR DESCRIPTION
**Pull request by joeds13**
Set a password policy, even though we have an SCP denying IAM user
creation, it stops false positive raising of this being insufficient
from Security Tests
Set the S3 Block Public Access controls at the account level, so we
don't need to set them individually on buckets

### CODE REVIEW

**Review by jamesnunnnhs — APPROVED** [review]

File: terraform/stacks/account/locals.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+locals {
+  name               = "list-reconciliation-${local.environment}"
+  environment        = lower(terraform.workspace)
+  data_clasification = local.environment == "prod" ? "5" : "1"
```
Comment (HarveyNHS) [inline terraform/stacks/account/locals.tf:4]: change clasification to classification

File: terraform/stacks/account/locals.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  tags = {
+    TagVersion         = "1"
+    Programme          = "SpinePod5"
+    Project            = "ListReconciliation"
+    DataClassification = local.data_clasification
```
Comment (HarveyNHS) [inline terraform/stacks/account/locals.tf:10]: same here

File: terraform/stacks/account/terraform.tf [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+terraform {
+  backend "s3" {
+    bucket         = "terraform-list-reconciliation-state-mgmt"
+    key            = "account.tfstate"
+    dynamodb_table = "terraform-list-reconciliation-locks"
+    region         = "eu-west-2"
+    encrypt        = true
+    role_arn       = "arn:aws:iam::486319732046:role/LRTerraformBase"
```
Comment (HarveyNHS) [inline terraform/stacks/account/terraform.tf:27]: Will this arn always be the same?

**Review by HarveyNHS — COMMENTED** [review]
Please review comments

File: terraform/stacks/account/terraform.tf (hunk 3 above)
Comment (joeds13) [reply terraform/stacks/account/terraform.tf:27]: Yeah, this role is in our single management account, which stores our Terraform state and locks

**Review by HarveyNHS — APPROVED** [review]
Approved

**Pull request merged**